rds/main.tf

```diff
@@ -0,0 +1,98 @@
+#lookup parameters for specic RDS types
+variable "ports" {
+  default = {
+    mysql = "3306"
+    oracle = "1521"
+  }
+}
+
+variable "families" {
+  default = {
+    mysql = "mysql5.6"
+    oracle = "oracle-se2-12.1"
+  }
+}
+
+variable "engines" {
+  default = {
+    mysql = "mysql"
+    oracle = "oracle-se2"
+  }
+}
+
+variable "engine_versions" {
+  default = {
+    mysql = "5.6.22"
+    oracle = "12.1.0.2.v2"
+  }
+}
+
+# Create RDS with Subnet and paramter group,
+resource "aws_security_group" "sg_rds" {
+  name = "sg_rds_${var.project}_${var.environment}${var.tag}"
+  description = "Security group that is needed for the RDS"
+  vpc_id = "${var.vpc_id}"
+
+  ingress {
+    from_port = "${lookup(var.ports, var.rds_type)}"
+    to_port = "${lookup(var.ports, var.rds_type)}"
+    protocol = "tcp"
+    security_groups  = ["${split(",", var.security_groups)}"]
+  }
+
+  tags {
+    Name = "${var.project}-${var.environment}${var.tag}-sg_rds"
+    Environment = "${var.environment}"
+    Project = "${var.project}"
+  }
+}
+
+resource "aws_db_subnet_group" "rds" {
+  name = "${var.project}-${var.environment}${var.tag}-rds"
+  description = "Our main group of subnets"
+  subnet_ids = ["${var.subnets}"]
+}
+
+resource "aws_db_parameter_group" "rds_mysql" {
+  name = "mysql-rds-${var.project}-${var.environment}${var.tag}"
```

Comment:
I would leave parameter groups out of the module, as these vary a lot depending on the project.